.github/workflows/create-deployment.yaml

```diff
@@ -0,0 +1,63 @@
+---
+name: Create deployment
+
+on:
+  pull_request:
+    branches:
+      - master
+      - release/*
+    types:
+      - labeled
+env:
+  GITOPS_ENVIRONMENT: stage
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    if: contains(github.event.pull_request.labels.*.name, 'deploy_nodes')
+
+    steps:
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.11
+
+      - name: Install Jinja2 cli
+        run: |
+          python -m pip install --upgrade pip
+          pip install jinja2-cli
+      - name: Checkout hoprnet
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Checkout gitops
+        uses: actions/checkout@v3
+        with:
+          repository: hoprnet/gitops
+          path: "./gitops"
+          token: ${{ secrets.COMMIT_BOT_ACCESS_TOKEN }}
+          ref: feature/deploy-nodes
+          fetch-depth: 0
+
+      - name: Set up environment
+        run: |
+          prBase="${{ github.event.pull_request.base.ref }}"
+          # The identification of the environment will be made once https://github.com/hoprnet/hoprnet/issues/4835 is finished
+          network=master-staging
```

Comment:
This should be configurable by the label, e.g. `staging-bratislava`, `staging-providence`...

Reply:
The idea is that every target branch (master , release/bratislava , release/xxxx) has an assigned `network` (monte_rosa, monte_rosa_2_0, etc). And eventually I will be able to get that from the releases.json or protocol-config.json. As this should change as per the issue mentioned (https://github.com/hoprnet/hoprnet/issues/4835), I left it uncompleted and limited this feature only for PR to `master`